src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,168 @@
+package duke;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import duke.task.Task;
+
+/**
+ * Handles the interactions with the user.
+ */
+public class Ui {
+    static final String CHATBOX_NAME = "Ado";
```

Comment:
Looks like you forgot to use or delete this constant.

Reply:
Oh, thanks for highlighting! :)